tools/dgeni/common/dgeni-definitions.ts

```diff
@@ -0,0 +1,33 @@
+import {ClassExportDoc} from 'dgeni-packages/typescript/api-doc-types/ClassExportDoc';
+import {ClassLikeExportDoc} from 'dgeni-packages/typescript/api-doc-types/ClassLikeExportDoc';
+import {PropertyMemberDoc} from 'dgeni-packages/typescript/api-doc-types/PropertyMemberDoc';
+import {NormalizedMethodMemberDoc} from './normalize-method-parameters';
+
+export interface CategorizedClassLikeDoc extends ClassLikeExportDoc {
```

Comment:
Add descriptions for these?

Reply:
Those are just simple modifications to the real Dgeni documents - added comments to all of the modified definitions in this file.